File: exercicios/Python/1038/1038.py
entrada = input().split();

cod = int(entrada[0])
qt = int(entrada[1])

if(cod == 1):
    vl = float(4.0 * qt)
elif(cod == 2):
    vl = float(4.5 * qt)
elif(cod == 3):
    vl = float(5 * qt)
elif(cod == 4):
    vl =float(2 * qt)
elif(cod == 5):
    vl = float(1.5 * qt)
print("Total: R$ {:.2F}".format(vl))

File: exercicios/JavaScript/1038/1038.js
var input = require('fs').readFileSync('stdin','utf8')
var lines = input.split('\n')

var line1 =  lines.shift().split(' ');

var op = {1:4.0, 2:4.5, 3:5.0, 4:2.0, 5:1.5}

var cod = parseInt(line1[0])
var qt = parseInt(line1[1])

var total = parseFloat(op[cod] * qt)

console.log("Total: R$ " + total.toFixed(2))